OSX pathing is incorrect on init install?

Works just fine on my windows machine. 
Unfortunately, tested on 2 Macs  with the same bad results.

**[Repo](https://github.com/morningharwood/platform)**

```
mharwood@mharwood-macbookpro: ~/Documents/GitHub/morningharwood/platform on master
$ tsc -v
Version 2.6.1


mharwood@mharwood-macbookpro: ~/Documents/GitHub/morningharwood/platform on master
$ yarn
✨  Done in 94.63s.

m@m: platform on master
$ npm start --app=portfolio

Cannot find module 'typescript' from '/Users/mharwood/Documents/GitHub'
Error: Cannot find module 'typescript' from '/Users/mharwood/Documents/GitHub'
    at Function.module.exports [as sync] (/Users/mharwood/Documents/GitHub/morningharwood/platform/node_modules/resolve/lib/sync.js:40:15)
    at Object.requireProjectModule (/Users/mharwood/Documents/GitHub/morningharwood/platform/node_modules/@angular/cli/utilities/require-project-module.js:6:28)
    at Object.readTsconfig (/Users/mharwood/Documents/GitHub/morningharwood/platform/node_modules/@angular/cli/utilities/read-tsconfig.js:6:48)
    at new NgCliWebpackConfig (/Users/mharwood/Documents/GitHub/morningharwood/platform/node_modules/@angular/cli/models/webpack-config.js:19:42)
    at Class.run (/Users/mharwood/Documents/GitHub/morningharwood/platform/node_modules/@angular/cli/tasks/serve.js:71:29)
    at check_port_1.checkPort.then.port (/Users/mharwood/Documents/GitHub/morningharwood/platform/node_modules/@angular/cli/commands/serve.js:123:26)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! morningharwood@0.0.0 start: `ng serve --app=portfolio`
```

weird directory to look for typescript.  For the lulz I adde it
```
mharwood@mharwood-macbookpro: ~/Documents/GitHub/morningharwood/platform
$ cd ../../
mharwood@mharwood-macbookpro: ~/Documents/GitHub
$ npm init -y

mharwood@mharwood-macbookpro: ~/Documents/GitHub
$ npm install --save typescript
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN github@1.0.0 No description
npm WARN github@1.0.0 No repository field.

+ typescript@2.6.1
added 1 package in 1.15s
```

Now we get a new error:
```
mharwood@mharwood-macbookpro: ~/Documents/GitHub/morningharwood/platform on master
$ npm start --app=portfolio

> morningharwood@0.0.0 start /Users/mharwood/Documents/GitHub/morningharwood/platform
> ng serve --app=portfolio

ENOENT: no such file or directory, stat '/Users/mharwood/Documents/GitHub/tsconfig.app.json'
Error: ENOENT: no such file or directory, stat '/Users/mharwood/Documents/GitHub/tsconfig.app.json'
```

**Why is the pathing so weird?**

--- a/packages/schematics/src/app/index.ts
+++ b/packages/schematics/src/app/index.ts
@@ -3,24 +3,23 @@ import {
   branchAndMerge,
   chain,
   externalSchematic,
+  filter,
+  MergeStrategy,
   mergeWith,
   move,
+  noop,
   Rule,
   template,
   Tree,
-  url,
-  MergeStrategy,
-  filter,
-  noop
+  url
 } from '@angular-devkit/schematics';
 import { Schema } from './schema';
 import * as stringUtils from '@schematics/angular/strings';
 import { addImportToModule, insert, toFileName } from '@nrwl/schematics';
-import * as path from 'path';
 import * as ts from 'typescript';
 import { addBootstrapToModule } from '@schematics/angular/utility/ast-utils';
 import { insertImport } from '@schematics/angular/utility/route-utils';
-import { serializeJson, addApp } from '../utility/fileutils';
+import { addApp, serializeJson } from '../utility/fileutils';
 import { addImportToTestBed } from '../utility/ast-utils';
 
 function addBootstrap(path: string): Rule {
@@ -143,7 +142,7 @@ export default function(schema: Schema): Rule {
       apply(url('./component-files'), [
         options.inlineTemplate ? filter(path => !path.endsWith('.html')) : noop(),
         template({ ...options, tmpl: '' }),
-        move(path.join(fullPath(options), 'app'))
+        move(`${fullPath(options)}/app`)
       ]),
       MergeStrategy.Overwrite
     ),
@@ -155,5 +154,5 @@ export default function(schema: Schema): Rule {
 }
 
 function fullPath(options: Schema) {
-  return path.join('apps', options.name, options.sourceDir);
+  return `apps/${options.name}/${options.sourceDir}`;
 }

--- a/packages/schematics/src/lib/index.ts
+++ b/packages/schematics/src/lib/index.ts
@@ -130,10 +130,9 @@ function updateTsLint(schema: Schema): Rule {
 
 export default function(schema: Schema): Rule {
   const options = { ...schema, name: toFileName(schema.name) };
-  const fullPath = path.join('libs', toFileName(options.name), options.sourceDir);
   const moduleFileName = `${toFileName(schema.name)}.module`;
-  const modulePath = path.join(fullPath, `${moduleFileName}.ts`);
-  const indexFile = path.join('libs', toFileName(options.name), 'index.ts');
+  const modulePath = `${fullPath(schema)}/${moduleFileName}.ts`;
+  const indexFile = `libs/${toFileName(options.name)}/index.ts`;
 
   if (schema.routing && schema.nomodule) {
     throw new Error(`nomodule and routing cannot be used together`);
@@ -165,5 +164,5 @@ export default function(schema: Schema): Rule {
 }
 
 function fullPath(options: Schema) {
-  return path.join('libs', options.name, options.sourceDir);
+  return `libs/${toFileName(options.name)}/${options.sourceDir}`;
 }

